packages/@apphosting/adapter-angular/src/bin/build.ts

```diff
@@ -1,20 +1,87 @@
 #! /usr/bin/env node
 import { spawn } from "child_process";
-import { loadConfig } from "../utils.js";
+import { loadConfig, ApplicationBuilderOptions, move, writeFile } from "../utils.js";
+import { resolve, normalize, relative } from "path";
+import { stringify as yamlStringify } from "yaml";
 
+interface OutputBundleOptions {
```

Comment:
Can we make an interfaces.ts and export this there?

Reply:
done